pkg/plugin/v3/scaffolds/webhook.go

```diff
@@ -19,13 +19,14 @@ package scaffolds
 import (
 	"fmt"
 
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/api"
```

Comment:
nit: do we want to group these imports with the other "sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/* imports in line 29?